common.yml

```diff
@@ -10,6 +10,8 @@
   roles:
     - greendayonfire.mongodb
 
+- include: tsuru_graphite.yml
+
```

Comment:
Is there a reason why this include appears between mongo and gandalf? Is it required by subsequent steps, or could it possibly be moved down to the end after all the standard tsuru components have been deployed (i.e. after router, but before post-install) ? - It would seem more logical in my mind that way as it's an add-on for use by tsuru rather than a core component.